src/pkg/services/m365/api/graph/service.go

```diff
@@ -416,7 +420,9 @@ func (aw *adapterWrap) Send(
 		err = stackWithCoreErr(ictx, err, 1)
 		e = err
 
-		if IsErrConnectionReset(err) || connectionEnded.Compare(err.Error()) {
+		if IsErrConnectionReset(err) ||
+			connectionEnded.Compare(err.Error()) ||
+			errors.Is(err, io.ErrUnexpectedEOF) {
```

Comment:
just double checking, this was an unexpected EOF error when sending the request, not when reading item data in kopia right? Not sure if you recall, but the latter would need something like the retry reader wrapper that we have

Reply:
This is while sending the request. e.g. we are seeing these errors while doing `GET /messages$expand=microsoft.graph.itemattachment%2Fitem`. Similarly seen for `getting mail attachment by id`.

Also, since we don't have KAI for exchange, all data reads are prefetched.

Reply:
>  since we don't have KAI for exchange, all data reads are prefetched.

🤦 excellent point!